src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Drawable, AreaCalculable {
+    private double radius;
+
+    public Circle(double radius, String color) {
+        super.setColor(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double area() {
+        return Math.PI * Math.pow(radius,2);
```

Comment:
Creating a figure, don't pass expressions in the constructor. Instead of calculating the area directly in the return statement, create a separate variable for the result of 'Math.pow(radius, 2)', and then use it in the return statement for better readability.